controllers/redirects/_form_create_toolbar.php

```diff
@@ -45,13 +29,3 @@
                     <?= e(trans('backend::lang.form.or')); ?> <a href="<?= Backend::url('winter/redirect/redirects'); ?>"><?= e(trans('backend::lang.form.cancel')); ?></a>
                 </span>
             </div>
-        </div>
-
-    <?= Form::close(); ?>
-
-<?php else: ?>
-
-    <p class="flash-message static error"><?= e($this->fatalError); ?></p>
-    <p><a href="<?= Backend::url('winter/redirect/redirects'); ?>" class="btn btn-default"><?= e(trans('winter.redirect::lang.redirect.return_to_redirects')); ?></a></p>
-
-<?php endif ?>
```

Comment:
Can we add the Create & New button to the default create partial and then remove this override entirely? We'll need to add the logic from the onCreate override in the RedirectsController to the FormController Behavior as well but I think it's worth having in the core.